sql/moz-fx-data-marketing-prod/google_search_console_derived/search_impressions_by_site_v1/metadata.yaml

```diff
@@ -3,23 +3,28 @@ description: |-
   Google Search impressions aggregated by site, synced by Fivetran to BigQuery for the following domains:
     * addons.mozilla.org
     * blog.mozilla.org
+    * developer.mozilla.org
     * getpocket.com
     * support.mozilla.org
     * www.mozilla.org
 
   Anonymized search queries aren't included.
 
-  We stopped syncing Google Search Console data with Fivetran in August 2023.
+  For the developer.mozilla.org domain, we stopped syncing Google Search Console data with Fivetran in May 2024.
+  See https://bugzilla.mozilla.org/show_bug.cgi?id=1890816.
+
+  For the other domains, we stopped syncing Google Search Console data with Fivetran in August 2023.
   See https://bugzilla.mozilla.org/show_bug.cgi?id=1764960#c44.
 owners:
 - srose@mozilla.com
 labels:
   incremental: true
   owner1: srose
-# Not scheduled because we stopped syncing Google Search Console data with Fivetran in August 2023.
-# See https://bugzilla.mozilla.org/show_bug.cgi?id=1764960#c44.
-#scheduling:
-#  date_partition_parameter: date
+scheduling:
+  # Not scheduled because we stopped syncing Google Search Console data with Fivetran.
+  # See https://bugzilla.mozilla.org/show_bug.cgi?id=1764960#c44 and https://bugzilla.mozilla.org/show_bug.cgi?id=1890816.
+  #dag_name: bqetl_google_search_console
+  date_partition_parameter: date
```

Comment:
With the custom `@date` parameter present in `query.sql`, if the `date_partition_parameter` metadata is commented out then some `bqetl` command(s) fail on that ETL because they fall back to assuming the parameter is `@submission_date`, but I forget which specific command(s) those were.

Reply:
I see, I think at least dry run uses it.  bqetl does use a mix of `date_partition_parameter` and `bigquery.time_partitioning`